Vagrantfile

```diff
@@ -116,8 +116,12 @@ Vagrant.configure(2) do |config|
         else
             # Ignore contrib/packaging/docker/stage to prevent concurrent
             # problems when using rsync on multiple VMs
+	    #
+	    # run rsync with options:
+	    #  -l: preserve symlinks (required for bazel symlinks)
+	    #  -c: Use checksum to prevent copying of files (avoid unnecessary bazel rebuilds)
             config.vm.synced_folder '.', '/home/vagrant/go/src/github.com/cilium/cilium', type: "rsync",
-            rsync__exclude: ["contrib/packaging/docker/stage", "src"]
+            rsync__exclude: ["contrib/packaging/docker/stage", "src"], rsync__args: "-lc"
```

Comment:
@ianvernon @eloycoto we probably need this for the test images?

Reply:
I'm trying out having bazel manage the Envoy dependency. This will remove the need for the linked files, so this should not be needed going forward.